app/models/chargeback.rb

```diff
@@ -29,7 +29,11 @@ def self.build_results_for_report_chargeback(options)
       data[key]["chargeback_rates"] = chargeback_rates.uniq.join(', ')
 
       # we are getting hash with metrics and costs for metrics defined for chargeback
-      data[key].calculate_costs(consumption, rates_to_apply)
+      if Settings[:new_chargeback]
```

Comment:
Is this accessing the global settings?  Would we set this in advanced settings for the server?

Reply:
Yes this work with advanced settings for server. So we can enable it here by line
```
:new_chargeback: false
```